hera_qm/firstcal_metrics.py

```diff
@@ -777,10 +742,9 @@ def plot_delays(self, ants=None, plot_type='both', cmap='nipy_spectral', ax=None
         if custom_ax is False:
             return fig
 
-    def plot_zscores(self, fname=None, plot_type='full', ax=None, figsize=(10, 6),
+    def plot_zscores(self, fname=None, plot_type='full', pol=None, ax=None, figsize=(10, 6),
```

Comment:
I agree with the rename of the class, but was not sure how many other things (possibly up/down stream could break) because of it. That being said, I _think_ the only place that calls it is an jupyter notebook so it probably wouldn't really matter.

I'm not very knowledgeable about why there are these kinds of "namespace pollution" as you call it, but I agree it would be better to write it out in a new PR.